lighthouse-core/test/audits/byte-efficiency/offscreen-images-test.js

```diff
@@ -109,16 +130,40 @@ describe('OffscreenImages audit', () => {
     const artifacts = {
       ViewportDimensions: DEFAULT_DIMENSIONS,
       ImageElements: [
-        // offscreen to the right
-        generateImage(generateSize(200, 200), [3000, 0], networkRecords[0]),
-        // offscreen to the bottom
-        generateImage(generateSize(100, 100), [0, 2000], networkRecords[1], url('B')),
-        // offscreen to the top-left
-        generateImage(generateSize(100, 100), [-2000, -1000], networkRecords[2], url('C')),
-        // offscreen to the bottom-right
-        generateImage(generateSize(100, 100), [3000, 2000], networkRecords[3], url('D')),
-        // half offscreen to the top, should not warn
-        generateImage(generateSize(1000, 1000), [0, -500], networkRecords[4], url('E')),
+        // Offscreen to the right.
+        generateImage({
+          size: generateSize(200, 200),
+          coords: [3000, 0],
+          networkRecord: networkRecords[0],
+        }),
+        // Offscreen to the bottom.
+        generateImage({
+          size: generateSize(100, 100),
+          coords: [0, 2000],
+          networkRecord: networkRecords[1],
+          src: url('B'),
+        }),
+        // Offscreen to the top-left.
+        generateImage({
+          size: generateSize(100, 100),
+          coords: [-2000, -1000],
```

Comment:
wdyt of making `.coords` be `.x` and `.y`?

Reply:
Sure! I don't have a particularly strong preference for `.coords`, so I can update it to `.x` and `.y`